.github/scripts/pull_legacy_checkpoints.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This file is meant to be used by community contributors as well, so would it be better to place this file in `scripts/` instead of in `.github/scripts/`?